plugins/modules/proxmox_kvm.py

```diff
@@ -800,6 +806,15 @@
     name: spynal
     node: sabrewulf
     revert: 'template,cpulimit'
+
+- name: Migrate VM on second node
+  community.general.proxmox_kvm:
+    api_user: root@pam
+    api_password: secret
+    api_host: helldorado
+    name: spynal
+    node: sabrewulf-2
+    migrate: True
```

Comment:
Sorry, I missed this one. YAML booleans must always be lower-case.
```suggestion
    migrate: true
```